modules/serverless-ossm-v1x.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * serverless/networking/serverless-ossm.adoc
+
+[id="serverless-ossm-v1x_{context}"]
+= Using {ProductShortName} 1.x with {ServerlessProductName}
+
+.Procedure
+
+. Add the namespaces that you would like to integrate with {ProductShortName} to the `ServiceMeshMemberRoll` object as members:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members:
+    - <namespace> <1>
+----
+<1> A list of namespaces to be integrated with {ProductShortName}.
++
+[IMPORTANT]
+====
+Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+====
+. Create a network policy that permits traffic flow from Knative system Pods to Knative services:
+.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving` namespace:
++
+[source,terminal]
+----
+$ oc label namespace knative-serving serving.knative.openshift.io/system-namespace=true
+----
+.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving-ingress` namespace:
++
+[source,terminal]
+----
+$ oc label namespace knative-serving-ingress serving.knative.openshift.io/system-namespace=true
+----
+.. For each namespace that you would like to integrate with {ProductShortName}, copy the following `NetworkPolicy` resource into a YAML file:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-from-serving-system-namespace
+  namespace: <namespace> <1>
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          serving.knative.openshift.io/system-namespace: "true"
+  podSelector: {}
+  policyTypes:
+  - Ingress
+----
+<1> A list of namespaces to be integrated with {ProductShortName}.
```

Comment:
This isn't a list. User needs to create same exact network policy for each namespace. As stated in line 42.

Reply:
I think actually the <1> in this case is redundant with the changes here, because we say to create for each namespace and have <namespace> as the variable.